src/app/loadout-builder/utils.ts

```diff
@@ -210,82 +139,25 @@ export function getFilteredPerksAndPlugSets(
   return { filteredPlugSetHashes, filteredPerks };
 }
 
-/** Whether this item is eligible for being in loadout builder */
+/** Whether this item is eligible for being in loadout builder. Now only armour 2.0 and only items that have all the stats. */
 export function isLoadoutBuilderItem(item: DimItem) {
   // Armor and Ghosts
-  return item.bucket.inArmor;
+  return (
```

Comment:
Moved the energy and stat filtering stuff here so we cut down as much stuff as possible, as early as possible.